broker/src/test/java/io/moquette/testclient/Client.java

```diff
@@ -52,6 +50,9 @@ public interface ICallback {
     }
 
     private static final Logger LOG = LoggerFactory.getLogger(Client.class);
+
+    private static final int TIMEOUT_DURATION_MS = 300;
```

Comment:
I would suggest to create a `Duration` and then change the usage:
```suggestion
    private static final Duration TIMEOUT_DURATION = Duration.ofMillis(300);
```